containers.go

```diff
@@ -444,7 +444,8 @@ func archiveFile(tf io.Writer, source io.Reader, header *tar.Header) error {
 			err := w.WriteHeader(&tar.Header{
 				Typeflag: tar.TypeDir,
 				Name:     strings.Join(parents[:i+1], "/"),
-				Mode:     0755,
+				// NOTE I'm not entirely sure, but this should be making /tmp not 1777, when it should
+				Mode: 0755,
```

Comment:
I wonder if this loop goes all the way up to `/tmp` and also changes its permissions bits :thinking: